app/assets/javascripts/components/features/compose/components/search.jsx

```diff
@@ -58,7 +58,7 @@ const Search = React.createClass({
 
         <div className='search__icon'>
           <i className={`fa fa-search ${hasValue ? '' : 'active'}`} />
-          <i className={`fa fa-times-circle ${hasValue ? 'active' : ''}`} onClick={this.handleClear} />
+          <i role='button' tabIndex='0' className={`fa fa-times-circle ${hasValue ? 'active' : ''}`} onClick={this.handleClear} />
```

Comment:
The role, tabIndex and onClick should really be on the div. The icon is content in this case.

And this element needs a label. Use `aria-label`.